src/main/java/it/gov/pagopa/payhub/activities/activity/ingestionflow/receipt/ReceiptIngestionActivityImpl.java

```diff
@@ -61,7 +61,7 @@ protected ReceiptIngestionFlowFileResult handleRetrievedFiles(List<Path> retriev
 
         try {
             ReceiptIngestionFlowFileResult result = csvService.readCsv(filePath, ReceiptIngestionFlowFileDTO.class, (csvIterator, readerExceptions) ->
-                    receiptProcessingService.processReceipts(csvIterator, readerExceptions, ingestionFlowFileDTO, workingDirectory));
+                    receiptProcessingService.processReceipts(csvIterator, readerExceptions, ingestionFlowFileDTO, workingDirectory), ingestionFlowFileDTO.getFileVersion());
```

Comment:
This flow has not version set on csv DTO (neither ingestionFlowFileDTO.getFileVersion() is probably set)
```suggestion
                    receiptProcessingService.processReceipts(csvIterator, readerExceptions, ingestionFlowFileDTO, workingDirectory), null);
```